spec/queue_it/api/event_spec.rb

```diff
@@ -142,17 +142,16 @@ module Api
             "EventStartTime"               => "2015-04-28T15:25:46.0000000Z",
             "EventEndTime"                 => "2015-04-28T19:25:46.0000000Z",
             "EventCulture"                 => event_culture_name,
-            "MaxNoOfRedirectsPrQueueId"    => "1",
+            "MaxNoOfRedirectsPrQueueId"    => "3",
             "QueueNumberValidityInMinutes" => "#{queue_number_validity_in_minutes}",
             "AfterEventLogic"              => "RedirectUsersToTargetPage",
             "AfterEventRedirectPage"       => "",
-            "UseSSL"                       => "Auto",
             "JavaScriptSupportEnabled"     => "False",
             "TargetUrlSupportEnabled"      => "True",
             "SafetyNetMode"                => "Disabled",
             "KnowUserSecurity"             => "MD5Hash",
             "KnowUserSecretKey"            => know_user_secret_key,
-            "CustomLayout"                 => "Default layout by Queue-it",
+            "CustomLayout"                 => "Dark theme",
```

Comment:
Is this a default theme by Queue-it? If yes, it does seem a bit daft that it isn't named `Dark theme by Queue-it`.

Also, should this be configurable? It seems like we're changing behavior.